app/controllers/items_controller.rb

```diff
@@ -10,11 +10,10 @@ def create
     @item.list_id = @list.id
     if authorized_user(@user) && @item.save
       flash[:notice] = "Item created successfully."
-      redirect_to group_list_path(@group, @list)
     else
       flash[:warning] = full_sentence_errors(@item)
-      render 'new'
     end
+    redirect_to group_list_path(@group, @list)
```

Comment:
@kmeyerhofer Change I made -- was getting a view not found error for `new` form for items. I assumed the desired behavior is to redirect back tot he path and display the appropriate error message